deps/wazuh_testing/wazuh_testing/vulnerability_detector.py

```diff
@@ -95,3 +101,101 @@ def make_vuln_callback(pattern):
     regex = re.compile(r'{}{}'.format(vulnerability_detector_prefix, pattern))
 
     return lambda line: regex.match(line) is not None
+
+
+def load_db(db_path):
+    conn = sqlite3.connect(db_path)
+    cursor = conn.cursor()
+    return conn, cursor
+
+
+def insert_package_in_db(agent = "000", format = "rpm", name = "WazuhIntegrationPackage", 
+    priority = "", section = "Unspecified", size= 99, vendor = "WazuhIntegrationTests", version = "1.0.0-1.el7",
+    architecture = "x86_64", multiarch = "", description = "Wazuh Integration tests mock package", 
+    source = "Wazuh Integration tests mock package", location = "", triaged = 0, cpe = "",
+    install_time = datetime.datetime.now().strftime("%Y/%m/%d %H:%M:%S")
+):
+    """
+    Insert a new package in the installed package database, with the parameters given as arguments.
+    If used in conjunction with insert_vulnerability using the default arguments it will generate an alert.
+    """
+    try:
+        path =  os.path.join(PACKAGES_DB_PATH,"{}.db".format(agent))
+        db = load_db(path)
+        sqlite_insert_query = """INSERT INTO sys_programs
+                            (scan_id, scan_time, format, name, priority, section, size, vendor, install_time, version, \
+                                architecture, multiarch, source, description, location, triaged, cpe)
+                            select scan_id, scan_time, \"{}\", \"{}\", \"{}\", \"{}\", \"{}\", \"{}\", 
+                            \"{}\", \"{}\", \"{}\",  \"{}\", \"{}\", \"{}\", \"{}\", \"{}\", \"{}\"
+                            from sys_programs limit 1
+                            """.format(
+                                format, name, priority, section, size, vendor, install_time, version, architecture, 
+                                multiarch, source, description, location, triaged, cpe
+                            )
+        db[1].execute(sqlite_insert_query)
+        db[0].commit()
+    except sqlite3.Error as error:
+        print("Failed to insert data into sqlite table\n", error)
+    finally:
+        db[1].close()
+        db[0].close()
+
+
+def insert_vulnerability(CVEID = "WVE-000", TARGET = "RHEL7", TARGET_MINOR= "", PENDING = 0, 
+    PACKAGE = "WazuhIntegrationPackage", OPERATION = "less than", OPERATION_VALUE = "2.0.0-1.el7",
+    TITLE = "", SEVERITY = "critical", PUBLISHED =  datetime.datetime.now().strftime("%Y-%m-%dT%H:%M:%SZ"),
+    UPDATED = "", REFERENCE = "https://github.com/wazuh/wazuh-qa", TARGET_V = "REDHAT", CVSS = "10.000000", 
+    CVSS_VECTOR = "AV:N/AC:L/Au:N/C:C/I:C/A:C", RATIONALE = "Wazuh integration test vulnerability", 
+    CVSS3 = "", BUGZILLA_REFERENCE = "https://github.com/wazuh/wazuh-qa", CWE = "WVE-000 -> WVE-001", 
+    ADVISORIES = "RHSA-2010:0029"
```

Comment:
Is better to use lowercase instead of uppercase in parameters definition to avoid any confusion with global or constant variables.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/700/commits/4d232de50ad20cca3108cd2423524a614fa93475